firestore-bigquery-export/guides/GENERATE_SCHEMA_VIEWS.md

```diff
@@ -360,7 +365,7 @@ ignored by the schema-views script.
 
 #### Cloud Firestore arrays
 
-Review [this example](https://github.com/firebase/extensions/blob/master/firestore-bigquery-export/guides/EXAMPLE_QUERIES.md) for querying an array.
+Review [this example](https://github.com/firebase/extensions/blob/master/firestore-bigquery-export/guides/EXAMPLE_QUERIES.md#example-queries-for-an-array) for querying an array.
```

Comment:
Is this where this link should probably go? @jhuleatt 
(also, pls double-check those headings in that example queries file, pls?)

Reply:
Yes, looks like that fits here!